src/prettytable/prettytable.py

```diff
@@ -1463,6 +1465,7 @@ def del_row(self, row_index):
                 f"table only has {len(self._rows)} rows"
             )
         del self._rows[row_index]
+        del self.dividers[row_index]
```

Comment:
This line isn't covered by tests. Please can you add a test case? Also for lines 1539 and 1547 below.